keps/sig-node/1769-memory-manager/README.md

```diff
@@ -586,20 +658,34 @@ the health of the service?**
     To understand the reason you will need to check via pod resources API 
     the amount of allocatable memory and memory reserved by containers.
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
   This does not seem relevant to this feature.
 
-* **Are there any missing metrics that would be useful to have to improve observability 
-of this feature?**
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
```

Comment:
I see some questions are left unanswered in this commit like this one, can we perhaps add a TODO to give an indication that they would be answered in subsequent commits.

Reply:
Done. Thanks.